src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -1479,3 +1493,174 @@ int xcldev::xclP2p(int argc, char *argv[])
 
     return ret;
 }
+
+/*
+ * m2mtest
+ */
+static void m2m_free_unmap_bo(xclDeviceHandle handle, unsigned boh, 
+auto * &boptr, size_t boSize)
```

Comment:
I think you need indent here. Shouldn't boptr just pass in as void *?

Reply:
execData is cast as "ert_start_copybo_cmd *" but the source and tgt bo are void* (I cast them as char*)